src/utils/package-info.ts

```diff
@@ -1,5 +1,11 @@
-import { join, resolve } from "path";
-let BridgeVersion: string;
+import { join, dirname } from "path";
+import { statSync } from "fs";
+import Logging from "../components/logging";
+
+const log = Logging.get("PackageInfo");
+
+// This may be defined if the script is run via NPM: https://docs.npmjs.com/cli/v8/using-npm/scripts#packagejson-vars
+let BridgeVersion: string|undefined = process.env.npm_package_version;
```

Comment:
Not easily, it's designed so that it's explicitly undefined if we've not found a version yet. `getBridgeVersion` will either use it if not defined, or fetch the version and then set it.